src/borg/crypto/key.py

```diff
@@ -749,6 +749,18 @@ class AuthenticatedKey(ID_BLAKE2b_256, RepoKey):
     ARG_NAME = 'authenticated'
     STORAGE = KeyBlobStorage.REPO
 
+    # It's only authenticated, not encrypted.
```

Comment:
ehrm. if we do not protect the key AND we have a repo key, how is that "authentication"?